requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
Just checking -- this app _is_ on python 3.9, right?

Reply:
yeah, it's running on python 3.9

Reply:
https://github.com/mitodl/mitxonline/blob/main/Dockerfile#L1